kf-app.deploy

```diff
@@ -3,7 +3,7 @@ deploy {
     architecture_type = "aws-ecs-service-type-1"
```

Comment:
Jenkins file: this file will be used by our _Jenkins_ server is _Kids-first_. Most likely, _Include_, in the near future, will have a similar file too.